modules/statics/src/coins.ts

```diff
@@ -2204,6 +2204,22 @@ export const coins = CoinMap.fromCoins([
     'TG3XXyExBkPp9nzdajDZsozEu4BkaSJozs',
     UnderlyingAsset['ttrx:usdt']
   ),
+  ttronToken(
+    '78551a7f-17d1-4e81-b6a9-3c6e960337f4',
+    'ttrx:usd1',
+    'Test USD1 Token',
+    18,
+    'TMsuh7QxbJkYaLsGMGCgNJYtYjTZLJG3Np',
+    UnderlyingAsset['ttrx:usd1']
+  ),
+  ttronToken(
+    '4853b203-e475-4538-bcba-f69da0e45d4b',
+    'ttrx:stgusd1',
+    'Test USD1 Token',
```

Comment:
ideally same name for 2 tokens shouldn't be an issue, but do you want to keep same name of both the tokens. Also the contracts are not yet deployed?

Reply:
https://shasta.tronscan.org/#/contract/TPEwuCefwvzBcr9BHvg2xhJCaf499uGMZx 

https://shasta.tronscan.org/#/contract/TMsuh7QxbJkYaLsGMGCgNJYtYjTZLJG3Np/code 

Both are already deployed, I added staging to one of the names